build/teamcity-nightly-roachtest.sh

```diff
@@ -4,6 +4,94 @@
 # apologies, you're going to have to dig around for them below or even better
 # yet, look at the job).
 
+# Note that when this script is called, the cockroach binary to be tested
+# already exists in the current directory.
+COCKROACH_BINARY="$PWD/cockroach.linux-2.6.32-gnu-amd64"
+if [[ ! -x ./cockroach ]]; then
```

Comment:
Why check for `./cockroach` and not `COCKROACH_BINARY`? Particularly when only `COCKROACH_BINARY` is used elsewhere but `./cockroach` doesn't seem to be used.

Reply:
Thanks for catching this, this is just wrong. I copied from Bob's comment.
The script isn't used yet, I will fix this and submit a PR with all of the other bugs that no doubt it contains when I've given this a test run.